### DISCUSSION TITLE
ESlint config

### DISCUSSION THREAD

**Discussion by Circlewee** [heading1/WYLSBingsu#24] [Polls]
# 주제
ESlint의 설정중에 여러 회사에서 오픈소스로 공개중인 유명한 것들이 존재합니다
- [Airbnb](https://github.com/airbnb/javascript)
- [Naver](https://github.com/naver/eslint-config-naver)  

공식문서를 읽으며 사용할 설정을 일일이 정의하는 것은 시간이 너무 오래걸릴것 같아 둘중 하나를 선택해 입맛에 맞게 수정하는 방법이 더 나을것 같다는 생각이 들어서 투표를 열어봅니다

> Naver도 Airbnb config를 기반으로 수정된 룰이기는 합니다.

### 설정 시 참고할만한 Reference
- [ESlint 설정](https://velog.io/@rimo09/ESLint-%EC%84%A4%EC%A0%95)
- [ESlint init](https://velog.io/@pandati0710/eslint-%EC%9D%B4%EA%B2%8C-%EB%8F%84%EB%8C%80%EC%B2%B4-%EB%AD%94%EB%8D%B0-%EB%82%A0-%EA%B4%B4%EB%A1%AD%ED%98%80)
- [ESLint, Prettier Airbnb style로 설정하기](https://velog.io/@_jouz_ryul/ESLint-Prettier-Airbnb-Style-Guide%EB%A1%9C-%EC%84%A4%EC%A0%95%ED%95%98%EA%B8%B0#12-eslint-airbnb-style-guid)

**Comment by sokojh** [comment]
Airbnb에서 기존 코드에 대해서 효율화를 진행하여 먼저 세상에 공개했었고,
그것이 개발자들의 열렬한 지지를 넘어서 비즈니스 회사들의 선택을 받았다면, 정말 필요한 것이 다 들어가 있다고 생각합니다.
Airbnb에 한표 살짝 해보겠습니다 ㅎㅎ 👀
